redaxo/src/addons/metainfo/lib/input/datetime.php

```diff
@@ -18,6 +21,16 @@ public function __construct()
         $this->timeInput = rex_input::factory('time');
     }
 
+    public function setStart($start)
```

Comment:
Hier vom naming her nicht eindeutig, oder?

Es könnte später auch ein start/end für die time-komponente geben...?

Reply:
Meine Überlegung war, dass eigentlich nur das Jahr ein Range braucht, konnte mir jetzt nicht vorstellen, dass man irgendwann vielleicht mal anbietet, dass nur Uhrzeiten zwischen 12:00 und 18:00 oder so möglich sein sollen.

Aber so oder so hast du Recht, dass es vom Naming her nicht eindeutig ist, und vermutlich ein anderer Name besser wäre. 
Also `setStartYear`? Drückt das korrekt aus, dass es um das Min-Jahr in der Jahr-Select-Box geht? Oder noch expliziter, `setYearRangeStart` oder so?

Param dann entsprechend auch umbenennen, oder? Also statt `start="1990"`  dann `start-year="1990"` o.ä. (müsste noch gucken ob wir an der stelle einen standard haben bzg. `-` vs. `_` vs. camelCase)

Reply:
> Meine Überlegung war, dass eigentlich nur das Jahr ein Range braucht, konnte mir jetzt nicht vorstellen, dass man irgendwann vielleicht mal anbietet, dass nur Uhrzeiten zwischen 12:00 und 18:00 oder so möglich sein sollen.

wenn man sowas wie die öffungszeiten eines ladengeschäfts oder sowas hat, ist oft nur von 8-17uhr o.ä.

> Also `setStartYear`? Drückt das korrekt aus, dass es um das Min-Jahr in der Jahr-Select-Box geht? Oder noch expliziter, `setYearRangeStart` oder so?

ich finde `setStartYear` passend.